pkg/data/management/role_data.go

```diff
@@ -140,7 +140,8 @@ func addRoles(wrangler *wrangler.Context, management *config.ManagementContext)
 		addRule().apiGroups("management.cattle.io").resources("clustermonitorgraphs").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("catalogtemplates").verbs("get", "list", "watch").
 		addRule().apiGroups("management.cattle.io").resources("catalogtemplateversions").verbs("get", "list", "watch").
-		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("get", "list", "watch")
+		addRule().apiGroups("catalog.cattle.io").resources("clusterrepos").verbs("get", "list", "watch").
+		addRule().apiGroups("management.cattle.io").resources("clusters").resourceNames("local").verbs("get")
```

Comment:
won't this give member of any cluster permission to get the local cluster? plus cluster-member could launch shell on 2.4 without this permission

Reply:
@mrajashree This will only impacts downstream cluster, see https://github.com/rancher/rancher/pull/29129#issuecomment-698053373